### DISCUSSION TITLE
Make Slow-motion FPS Independent of Anti-Flicker Region

### DISCUSSION THREAD

**Discussion by AlexLandherr** [gopro/labs#554] [Feature Requests]
This is similar to an earlier feature request but concerns slow-mo filming.

I would like to make it possible to shoot in 240 fps at 2.7K regardless of whether my anti-flicker region is NTSC or PAL.
Or put another way: all slow-mo shooting modes should be available to all anti-flicker regions.

Apparently this was possible a few years back but was removed for some reason.

**Comment by dnewman-gpsw** [comment]
This is not something Labs can do, is it can't change where modes are listed. Of course, this is just an inconvenience as scanning the command p240 will get you 240fps, automatically switching the anti-flicker as needed.

**Reply by AlexLandherr** [reply]
Ok, just checking. Thanks for clearing things up.
